cluster/saltbase/salt/monit/kube-proxy

```diff
@@ -3,7 +3,9 @@ group kube-proxy
 start program = "/etc/init.d/kube-proxy start"
 stop program = "/etc/init.d/kube-proxy stop"
 if does not exist then restart
-if failed port 10249
-  protocol HTTP request "/healthz"
-  with timeout 10 seconds
-  then restart
+if failed
```

Comment:
The default is 5 seconds (which is what I was going to lower this to). If we can make an HTTP connection to localhost in 5 seconds something is really wrong. Also note that we were already using the default timeout in some of the other monit config files.